src/wesnoth.cpp

```diff
@@ -1158,6 +1158,21 @@ int main(int argc, char** argv)
 				}
 				game_config::path = std::move(auto_dir);
 			}
+			else {
+				bool data_dir_specified = false;
+				for(int i=0;i<argc;i++)
+				{
```

Comment:
Coding standard and surrounding code both put the opening brace on the same line as the conditional.